<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>261</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Araç Türü</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Marka</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Vites</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Yakıt Türü</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Üretim Yılı</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Motor Gücü</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Kasa Tipi</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEditTur"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditMarka"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditModel"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditVites"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditYakit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditUretimYili"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditMotorGucu"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditKasaTipi"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color:rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>Araç Ekle</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>271</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Motor Hacmi</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Kapı</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Renk</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Motor No</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Şasi No</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Günlük Ücret</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Kirada Mı ?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color:rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Kullanım Dışı</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEditMotorHacmi"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditKapi"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditRenk"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditMotorNo"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditSasiNo"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditGunlukBedel"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Kaydet</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
